packages/components/molecules/f-rating/src/components/_tests/Rating.test.js

```diff
@@ -1,10 +1,117 @@
-import { shallowMount } from '@vue/test-utils';
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import { VueI18n } from '@justeat/f-globalisation';
 import VRating from '../Rating.vue';
+import i18n from './helpers/setup';
+
+const localVue = createLocalVue();
+localVue.use(VueI18n);
 
 describe('Rating', () => {
+    let propsData;
+    let wrapper;
+    const $tc = jest.fn();
+
+    beforeEach(() => {
+        propsData = {
+            starRating: 2,
+            maxStarRating: 5
```

Comment:
if we're not going to define these on a test by test basis, I'd declare them outside the beforeEach so tests can access them.

That way we can add a test to check that when the starRating provided is equal to maxStarRating, it behaves correctly.